squad/frontend/views.py

```diff
@@ -571,6 +571,12 @@ def build_attachment(request, group_slug, project_slug, build_version, testrun,
     return attachment(request, group_slug, project_slug, build_version, testrun, None, None, filename)
 
 
+@auth
+def download_attachment(request, group_slug, project_slug, build_version, attachment_id):
```

Comment:
this is only slightly different from attachment() function. Maybe try patching it instead of creating a new one?

Reply:
@mwasilew agree..changed this a bit.